.github/workflows/merge.yml

```diff
@@ -0,0 +1,110 @@
+name: Various autofixes on the code
+
+# only trigger on pull request closed events
+on:
+  pull_request:
+    types: [ closed ]
+
+jobs:
+
+  code_format_commit:
+    if: github.event.pull_request.merged == true
+    name: Style/format
+    runs-on: ${{ matrix.job.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+          - { os: ubuntu-latest , features: feat_os_unix }
+    steps:
+    - uses: actions/checkout@v2
+    - name: Initialize workflow variables
+      id: vars
+      shell: bash
+      run: |
+        ## VARs setup
+        # target-specific options
+        # * CARGO_FEATURES_OPTION
+        CARGO_FEATURES_OPTION='' ;
+        if [ -n "${{ matrix.job.features }}" ]; then CARGO_FEATURES_OPTION='--features "${{ matrix.job.features }}"' ; fi
+        echo set-output name=CARGO_FEATURES_OPTION::${CARGO_FEATURES_OPTION}
+        echo ::set-output name=CARGO_FEATURES_OPTION::${CARGO_FEATURES_OPTION}
+    - name: Install `rust` toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        default: true
+        profile: minimal # minimal component installation (ie, no documentation)
+        components: rustfmt
+    - name: "`cargo fmt` run"
+      shell: bash
+      run: |
+        cargo fmt
+    - name: "`fmt` testing of tests"
+      shell: bash
+      run: |
+        # `cargo fmt` of tests
+        find tests -name "*.rs" -print0 | xargs -0 cargo fmt --
+    - name: Commit rustfmt changes
+      uses: EndBug/add-and-commit@v7
+      with:
+        default_author: github_actions
+        message: "maint ~ rustfmt (`cargo fmt`)"
+
+
+  min_version:
+    name: MinRustV # Minimum supported rust version
```

Comment:
I think a version of this step should also stay in 'CICD', maybe without the 'Cargo.lock' test to obviously show any non-MinRustV compatible blocker and log dependency info for PR iterations. 

`RUST_MIN_SRV` will also need to be set in the env here as well (could it be derived easily from 'CICD.yml' with a grep? and pushed as a step output, so as to keep it in one location?).

Reply:
hmm, not sure how to do it right now! I will have to get back on this later (don't hesitate to steal it if you know :)

Reply:
Sure, I'll look at it.